subsys/net/l2/openthread/openthread.c

```diff
@@ -326,10 +326,17 @@ int openthread_send(struct net_if *iface, struct net_pkt *pkt)
 int openthread_start(struct openthread_context *ot_context)
 {
 	otInstance *ot_instance = ot_context->instance;
-	otError error;
+	otError error = OT_ERROR_NONE;
 
 	openthread_api_mutex_lock(ot_context);
 
+	NET_INFO("OpenThread version: %s", otGetVersionString());
+
+	if (!IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR)) {
```

Comment:
This shouldn't be negated: `if (IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR)) {`

Reply:
done.